src/components/DetailedInfoArrea/styles.tsx

```diff
@@ -0,0 +1,106 @@
+import styled from "styled-components"
```

Comment:
change the extension of the file from 'tsx' to 'ts', cause we don't have layout here